host/posix_u_wrapper.c

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+
+#include <openenclave/edger8r/host.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/posix.h>
+#include "hostthread.h"
+
+#if defined(_MSC_VER)
+#define st_atime st_atime
+#define st_mtime st_mtime
+#define st_ctime st_ctime
+#endif
+
+/* Override oe_call_enclave_function() calls with _call_enclave_function(). */
+#define oe_call_enclave_function _call_enclave_function
```

Comment:
@mikbras @andschwa 
I implemented a similar functionality in the branch: https://github.com/Microsoft/openenclave/tree/anakrish/edger8r/internal.
@dthaler had very similar requirements for trustzone/optee of generating marshaling code for internal libraries (sockets, files etc).
My recommendation would be the accept the implementation of "internal" EDL libraries as in this PR with the understanding that we enhance oeedger8r to support internal libraries as first class concept. At that point, we will rework this PR's way of implementing internal libraries.

Reply:
Yes, I agree. I'm amenable to a better design later on.